libs/blocks/chart/chart.js

```diff
@@ -0,0 +1,72 @@
+import { makeRelative } from '../../utils/utils.js';
+
+function processDataset(data) {
+  const dataset = {};
+  const header = Object.keys(data[0]);
+  
+  // Remove group and unit from header
+  const unit = header.indexOf('Unit');
+  if (unit !== -1) header.splice(unit, 1);
+  const group = header.indexOf('Group');
+  if (group !== -1) header.splice(group, 1);
```

Comment:
nit: since `unit` and `group` are only used for the `if`s, inline them, also `includes` is cleaner.

Will the header strings always be capitalized?  Should it be a case-insensitive check?

```suggestion
  if (header.includes('Unit')) header.splice(unit, 1);
  if (header.includes('Group')) header.splice(group, 1);
```

Reply:
Yes, it probably should be case-insensitive. I understand your comment but we still need the index for `splice`, I could use `filter` to remove group and unit without needing an index while being case-insensitive.
~~~
  let header = Object.keys(data[0]);
  // Remove group and unit from header
  header = header.filter(title => title.toLowerCase() !== 'unit');
  header = header.filter(title => title.toLowerCase() !== 'group');
~~~

Reply:
doh!  Filter is def is the way to go.